apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/FirestoreDbTest.cs

```diff
@@ -283,7 +283,31 @@ public async Task GetDocumentSnapshotsAsync_WithTransaction()
             };
             mock.Setup(c => c.BatchGetDocuments(request, It.IsAny<CallSettings>())).Returns(new FakeDocumentStream(responses));
             var db = FirestoreDb.Create("proj", "db", mock.Object);
-            var results = await db.GetDocumentSnapshotsAsync(new[] { db.Document("col1/doc1") }, transactionId, default);
+            var results = await db.GetDocumentSnapshotsAsync(new[] { db.Document("col1/doc1") }, transactionId, fieldMask: null, default);
+            mock.VerifyAll();
+        }
+
+        [Fact]
+        public async Task GetDocumentSnapshotsAsync_WithFieldMask()
+        {
+            // This test is about checking the transaction is copied...
```

Comment:
This comment here is probably from copy/paste.

Reply:
Yup! Will delete the transaction bits.